doc/user/zebra.rst

```diff
@@ -58,6 +58,26 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
 
 .. _interface-commands:
 
+Configuration Addresses behaviour
+=================================
+
+At startup, when running *Zebra*, the daemon will first discover the
+underlying networking objects from the operating system. This includes
+interfaces, addresses of interfaces, static routes. Then, it will exec
+the configuration file, including its own addressing scheme. All this
```

Comment:
The meaning of `addressing scheme` is not clear to me.

Reply:
s/addressing scheme/interface addresses, static routes, etc